client/src/inside/layout/LayoutEhr.vue

```diff
@@ -13,10 +13,10 @@
               ehr-banner
           div(class="ehr_layout__content_page")
             slot Main EHR content for a component will appear here. The component is selected by the router
-      app-footer
-      input(class="checkbox", type="checkbox", v-model="showingSpecial")
-      div(v-show="showingSpecial")
-        ehr-special
+    input(class="checkbox", type="checkbox", v-model="showingSpecial")
+    div(v-show="showingSpecial")
+      ehr-special
+    app-footer
```

Comment:
Need to keep that input below the footer.
Don't make any changes for now.  We can un-fix this in another PR.